### DISCUSSION TITLE
How to see live logs > fm logs example.localhost --follow

### DISCUSSION THREAD

**Discussion by kittiu** [rtCamp/Frappe-Manager#126] [Q&A] [answered]
Hello, first of all, thank you for creating this awesome dev env. 

I want to see the log by using this command
```
$ fm logs test.localhost --follow --service frappe
```
But I only get this,
```
2024-03-04 06:25:16,903 INFO RPC interface 'supervisor' initialized
2024-03-04 06:25:16,903 CRIT Server 'unix_http_server' running without any HTTP authentication checking
2024-03-04 06:25:16,903 INFO supervisord started with pid 115
2024-03-04 06:25:17,905 INFO spawned: 'bench-dev' with pid 129
2024-03-04 06:25:22,688 INFO success: bench-dev entered RUNNING state, process has stayed up for > than 4 seconds (startsecs)
::: Showing logs
```

I test by tapping the x = 1/0 on save of a document. But nothing here move. I can only see the log from the file `bench-start.log`
i.e.,
```
06:28:13 web.1      |   File "apps/frappe/frappe/model/document.py", line 952, in fn
06:28:13 web.1      |     return method_object(*args, **kwargs)
06:28:13 web.1      |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
06:28:13 web.1      |   File "apps/erpnext/erpnext/selling/doctype/sales_order/sales_order.py", line 194, in validate
06:28:13 web.1      |     x = 1/0
06:28:13 web.1      |         ~^~
06:28:13 web.1      | ZeroDivisionError: division by zero
```

How can I see the live log?

**Comment by Xieyt** [comment]
Hey @kittiu,

To display and follow the frappe dev server logs, you can utilize the following command:

```bash
$ fm logs test.localhost --follow
```
Alternatively, you can tail the file `frappe-bench/logs/web.dev.log` within the respective `workspace` directory.

**Reply by kittiu** [reply]
I did try exaxtly as you mentioned, `$ fm logs test.localhost --follow` but I only get stuck here, log is not running.

![image](https://github.com/rtCamp/Frappe-Manager/assets/1973598/73a40022-c405-4649-a80d-71a0794c5e9b)

And for the log file, I have this one `frappe-bench/logs/bench-start.log` I don't see the file you mentioned `web.dev.log`

**Reply by Xieyt** [reply, marked as answer]
Can you let me know which version of `fm` you are using  ? ( You can find it using `fm --version` ). If it's not `0.11.0`, please update it using `pip install --upgrade frappe-manager`.

**Reply by kittiu** [reply]
Yes, my version was old, and the new version works as expected.
Thanks!
